internal/dinosaur/pkg/cmd/fleetmanagerclient/client.go

```diff
@@ -0,0 +1,61 @@
+// Package fleetmanagerclient is a client for the CLI to connect to the fleetmanager.
+package fleetmanagerclient
+
+import (
+	"os"
+	"sync"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/stackrox/acs-fleet-manager/pkg/client/fleetmanager"
+)
+
+var (
+	singletonInstance sync.Once
+	client            *fleetmanager.Client
+)
+
+const (
+	defaultFleetManagerEndpoint = "http://localhost:8000"
+	fleetManagerEndpointEnvVar  = "FMCLI_FLEET_MANAGER_ENDPOINT"
+	ocmRefreshTokenEnvVar       = "OCM_TOKEN"
+)
+
+// AuthenticatedClientWithOCM returns a rest client to the fleet-manager and receives the OCM refresh token.
+// This function will panic on an error, designed to be used by the fleet-manager CLI.
+func AuthenticatedClientWithOCM() *fleetmanager.Client {
+	ocmRefreshToken := os.Getenv(ocmRefreshTokenEnvVar)
+	if ocmRefreshToken == "" {
+		panic("OCM_TOKEN not set. Please set OCM token with 'export OCM_TOKEN=$(ocm token --refresh)'")
```

Comment:
nit: use the const to build the error 
```suggestion
		panic(ocmRefreshTokenEnvVar + " not set. Please set OCM token with 'export " + ocmRefreshTokenEnvVar + "=$(ocm token --refresh)'")
```

Reply:
done